quickstart.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+
+# Exposing ports
+EXPOSE 6900
```

Comment:
It could be also interesting expose the default elasticsearch port (9200)

Reply:
Sure I'll add that